### ISSUE TITLE
[Spellque] Spellque can sometimes trigger gcd from spells that are off-GCD

### ISSUE THREAD

**Issue by Aeded** [Atlantiss/NetherwingBugtracker#8251]
**Description**: 
For example, if you are casting already casting a spell and que Elemental Mastery+Spell (Chain/bolt/shock) it can happen that you only cast Elemental Mastery and recieve GCD without casting the second spell.

**Current behaviour**:
https://streamable.com/up3e3y
here I spellqued macro that had Elemental Mastery+Chain Lightning

**Expected behaviour**: 
You should be able to spellque off-GCD spells in combination with other spells without recieving GCD
**Server Revision**: 3719

**Comment by Wolffenstein** [comment]
Spell Queue doesn't support queuing more than one spell

**Comment by Aeded** [comment]
So @Wolffenstein  you saying its blizzlike that you get GCD from a spell that isnt supposed to trigger gcd?
You have it implemented wrong, if anything it should just que the last spell in the macro and skip all previous, but this can be handled by /cqs at first line which makes interaction I described possible. however /cqs doesnt apparently work here.
nvm, cqs was added in cata, however you are still not supposed to get GCD for nothing
